src/store/actions/Chat.ts

```diff
@@ -37,12 +44,12 @@ export const setVenueChatTabOpened = (): SetVenueChatTabOpenedAction => ({
 });
 
 export const setPrivateChatTabOpened = (
-  recipientId?: string
+  recipient: WithId<ChatUser> | undefined
```

Comment:
```suggestion
  recipient?: WithId<ChatUser>
```

Reply:
the same https://github.com/sparkletown/sparkle/pull/2304#discussion_r711879752